### PR TITLE
Smr post

### PR DESCRIPTION
**Pull request by steve-at-hx**
### Blog title
Selling the dream at SMR16

### Blog description 
Overview of our experience at SMR16 and the challenge in selling something that's actually very good.

#### Author
Steve CS
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by DuncanFenning** [comment]
I would be honoured for Editor 1️⃣

**Comment by carolinewoodhams** [comment]
Taking "2"

**Comment by carolinewoodhams** [comment]
Steve - could you update my twitter handle as well if you're changing that file? It should now be cwoodhams82 - thanks!

File: _posts/2016-05-12-smr16.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Selling the dream at SMR16
+categories: Conferences, Recruitment, SMR, Life
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: Just to note this here, not that this needs changing right now (need some time to think about it) but we've got to ensure that the categories & tags are working for us - to optimise our SEO. Categories are like the 'contents' in a book, and tags are the 'index'. I'll be undertaking a review of recent posts to see if we can improve this & put together some guidelines going forwards.

File: _posts/2016-05-12-smr16.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+excerpt: The Hungry Geek took part in the Silicon Milkroundabout 2016 tech job fair this May. We faced some stiff competition from other companies also looking for the best and brightest to join us. Here's how we _faired_. Get it?
+---
+
+On Sunday 5th May, our Hungry Geeks attended the [Silicon Milkroundabout 2016](https://www.siliconmilkroundabout.com/) to share our [great job opportunities at Holiday Extras](http://join.holidayextras.co.uk/vacancies/) with some of the best and brightest tech job seekers. 
+
+Finding candidates is tough! One of the great things about SMR is that the vast majority of attendees are actively looking, but because it’s such a good opportunity to engage this group, it’s very competitive. Standing out from a crowd that includes some big, recognised names is a challenge.
+
+### Standing out from the crowd.
+
+Our People team did an amazing job of providing our Geeks with a great platform from which to promote HX. Between our head turning signage to our alluring swag we had a lot of visitors. The stand gave a great narrative of our surroundings, our ethos, our tech stack and our industry recognition.
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: include hyphen - head-turning

File: _posts/2016-05-12-smr16.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+![Photo of our SMR stand](https://github.com/holidayextras/holidayextras.github.com/blob/smr_post/assets/img/IMG_1805.JPG "Our SMR stand")
+
+We also made sure that our guys on the stand were part of the teams we are recruiting to. What we lacked in confidence in approaching strangers and recruitment lingo we made up for in real, applied knowledge of our tech stack and what it’s like to be a Hungry Geek at HX.
+
+We have to try hard at these events because we’re competing against big brands to get noticed. For example, the large supermarket chain next to us (not in the Times 100 Best Companies to Work For or particularly known for tech careers) rocked up with little more than a couple of pull ups but still had great foot traffic, perhaps at least in part because nearly every attendee would have passed their branding on the way to the event that day. 
+
+Still, we try. We try because we know we have great opportunities for excellent candidates. The job is not complete with an attractive stand, though, as we’re not the only people in our position going all out to be recognised.
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: "we try" sounds a little desperate. How about "we did our best" - which provides an opportunity to link into our value of be the best version of yourself?

File: _posts/2016-05-12-smr16.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+We have to try hard at these events because we’re competing against big brands to get noticed. For example, the large supermarket chain next to us (not in the Times 100 Best Companies to Work For or particularly known for tech careers) rocked up with little more than a couple of pull ups but still had great foot traffic, perhaps at least in part because nearly every attendee would have passed their branding on the way to the event that day. 
+
+Still, we try. We try because we know we have great opportunities for excellent candidates. The job is not complete with an attractive stand, though, as we’re not the only people in our position going all out to be recognised. 
+
+### No, really… It’s that good.
+
+I can only comment on my own experience, but I have known companies who in an effort to stand out promote beyond their means and are not able to deliver on their promise. I would be extremely surprised if you hadn’t too. Ever worked somewhere when work life balance became condensed into a Sunday afternoon? I can count myself as a skeptic on this front and wouldn’t consider myself alone in this position.
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: "work-life" (needs a hyphen!)

File: _posts/2016-05-12-smr16.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+However, the biggest surprise since I joined HX in March 2016 is that you can believe the hype. I feel absolutely entertained, rewarded and well looked after, as promised. Ultimately, I feel honoured to be here. One of the hardest things then, in relation to recruiting, is to stay cool and not gush to the point of leading a skeptic to think we’re trying hard for the wrong reasons. Apologies to anyone who received an unstructured rambling of superlatives.
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: maybe add emphasis on "can" - this is a really bold statement :)

Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: Last sentence - applicable to conversations on the day or this particular paragraph? ;)

File: _posts/2016-05-12-smr16.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+### Here’s hoping!
```
Comment (carolinewoodhams) [inline _posts/2016-05-12-smr16.md]: Again, sounds a little desperate. This part is about the slow burn of getting a brand recognised - could the challenge be phrased more confidently/positively?

e.g. "slowly slowly catchy monkey"

**Comment by carolinewoodhams** [comment]
Really great post - love your style of writing, Steve. Just a few suggestions, hope they help! 

Also - can we caption photos? This will then count towards SEO rankings too. My HG blog knowledge is limited, I'm not sure how to do this but a dev might ;)

**Comment by steve-at-hx** [comment]
Thank you for your comments, Caroline. I've gone through them this morning and addressed each one I hope. Easiest place to browse the copy changes is here: https://github.com/holidayextras/holidayextras.github.com/pull/98/commits/ad084727a592fc686ad722e3135c6c1af90d7783

File: _posts/2016-05-12-smr16.md (hunk 2 above)
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: `Between our head turning signage to our alluring swag we had a lot of visitors` doesn't quite sound right to me.

Would `Between our head turning signage and our alluring swag we had a lot of visitors` sound better?

File: _posts/2016-05-12-smr16.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+excerpt: The Hungry Geek took part in the Silicon Milkroundabout 2016 tech job fair this May. We faced some stiff competition from other companies also looking for the best and brightest to join us. Here's how we _faired_. Get it?
+---
+
+On Sunday 5th May, our Hungry Geeks attended the [Silicon Milkroundabout 2016](https://www.siliconmilkroundabout.com/) to share our [great job opportunities at Holiday Extras](http://join.holidayextras.co.uk/vacancies/) with some of the best and brightest tech job seekers. 
+
+Finding candidates is tough! One of the great things about SMR is that the vast majority of attendees are actively looking, but because it’s such a good opportunity to engage this group, it’s very competitive. Standing out from a crowd that includes some big, recognised names is a challenge.
+
+### Standing out from the crowd.
+
+Our People team did an amazing job of providing our Geeks with a great platform from which to promote HX. Between our head-turning signage to our alluring swag we had a lot of visitors. The stand gave a great narrative of our surroundings, our ethos, our tech stack and our industry recognition.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: `Between our head turning signage to our alluring swag we had a lot of visitors` doesn't quite sound right to me.

Would `Between our head turning signage and our alluring swag we had a lot of visitors` or `From our head turning signage to our alluring swag we had a lot of visitors` sound better?

File: _posts/2016-05-12-smr16.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+Finding candidates is tough! One of the great things about SMR is that the vast majority of attendees are actively looking, but because it’s such a good opportunity to engage this group, it’s very competitive. Standing out from a crowd that includes some big, recognised names is a challenge.
+
+### Standing out from the crowd.
+
+Our People team did an amazing job of providing our Geeks with a great platform from which to promote HX. Between our head-turning signage to our alluring swag we had a lot of visitors. The stand gave a great narrative of our surroundings, our ethos, our tech stack and our industry recognition. 
+
+![Photo of our SMR stand](https://github.com/holidayextras/holidayextras.github.com/blob/smr_post/assets/img/IMG_1805.JPG "Our SMR stand")
+
+We also made sure that our guys on the stand were part of the teams we are recruiting to. What we lacked in confidence in approaching strangers and recruitment lingo we made up for in real, applied knowledge of our tech stack and what it’s like to be a Hungry Geek at HX.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: Could this be `... recruiting for` or `recruiting in`?

File: _posts/2016-05-12-smr16.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+Finding candidates is tough! One of the great things about SMR is that the vast majority of attendees are actively looking, but because it’s such a good opportunity to engage this group, it’s very competitive. Standing out from a crowd that includes some big, recognised names is a challenge.
+
+### Standing out from the crowd.
+
+Our People team did an amazing job of providing our Geeks with a great platform from which to promote HX. Between our head-turning signage and our alluring swag we had a lot of visitors. The stand gave a great narrative of our surroundings, our ethos, our tech stack and our industry recognition. 
+
+![Photo of our SMR stand](https://github.com/holidayextras/holidayextras.github.com/blob/smr_post/assets/img/IMG_1805.JPG "Our SMR stand")
+
+We also made sure that our guys on the stand were part of the teams we are recruiting to. What we lacked in confidence in approaching strangers and recruitment lingo we made up for in real, applied knowledge of our tech stack and what it’s like to be a Hungry Geek at HX.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: Perhaps `recruiting for` or `recruiting in` reads better?
Also does `What we lacked in confidence regarding approaching strangers and recruitment lingo` sound better?

File: _posts/2016-05-12-smr16.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+We have to try hard at these events because we’re competing against big brands to get noticed. For example, the large supermarket chain next to us (not in the Times 100 Best Companies to Work For or particularly known for tech careers) rocked up with little more than a couple of pull ups but still had great foot traffic, perhaps at least in part because nearly every attendee would have passed their branding on the way to the event that day.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: Could this be sentence be broken down? I tend to find smaller sentences read better. Just my opinion however

File: _posts/2016-05-12-smr16.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+![Photo of our SMR stand](https://github.com/holidayextras/holidayextras.github.com/blob/smr_post/assets/img/IMG_1805.JPG "Our SMR stand")
+
+We also made sure that our guys on the stand were part of the teams we are recruiting to. What we lacked in confidence in approaching strangers and recruitment lingo we made up for in real, applied knowledge of our tech stack and what it’s like to be a Hungry Geek at HX.
+
+We have to try hard at these events because we’re competing against big brands to get noticed. For example, the large supermarket chain next to us (not in the Times 100 Best Companies to Work For or particularly known for tech careers) rocked up with little more than a couple of pull ups but still had great foot traffic, perhaps at least in part because nearly every attendee would have passed their branding on the way to the event that day. 
+
+Still, we go all out. We go all out because we know we have great opportunities for excellent candidates. The job is not complete with an attractive stand, though, as we’re not the only people in our position going all out to be recognised.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: I'm not the best at punctuation but I'm not sure the double commas are necessary around `though`

File: _posts/2016-05-12-smr16.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+![Photo of our SMR stand](https://github.com/holidayextras/holidayextras.github.com/blob/smr_post/assets/img/IMG_1805.JPG "Our SMR stand")
+
+We also made sure that our guys on the stand were part of the teams we are recruiting to. What we lacked in confidence in approaching strangers and recruitment lingo we made up for in real, applied knowledge of our tech stack and what it’s like to be a Hungry Geek at HX.
+
+We have to try hard at these events because we’re competing against big brands to get noticed. For example, the large supermarket chain next to us (not in the Times 100 Best Companies to Work For or particularly known for tech careers) rocked up with little more than a couple of pull ups but still had great foot traffic, perhaps at least in part because nearly every attendee would have passed their branding on the way to the event that day. 
+
+Still, we go all out. We go all out because we know we have great opportunities for excellent candidates. The job is not complete with an attractive stand, though, as we’re not the only people in our position going all out to be recognised. 
+
+### No, really… It’s that good.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: lower case `it's`?

File: _posts/2016-05-12-smr16.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+I can only comment on my own experience, but I have known companies who in an effort to stand out promote beyond their means and are not able to deliver on their promise. I would be extremely surprised if you hadn’t too. Ever worked somewhere when work-life balance became condensed into a Sunday afternoon? I can count myself as a skeptic on this front and wouldn’t consider myself alone in this position.
```
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: Might want a comma here somewhere. Perhaps after `who` and `stand out`

File: _posts/2016-05-12-smr16.md (hunk 9 above)
Comment (steve-at-hx) [inline _posts/2016-05-12-smr16.md]: Sorry Duncan, I see what you're saying, but I think I prefer the way it's written now on this one.

File: _posts/2016-05-12-smr16.md (hunk 13 above)
Comment (steve-at-hx) [inline _posts/2016-05-12-smr16.md]: See what you're saying, but I'm in danger of putting in too many commas I think. If the sentences around it were less commary, I'd make the change.

File: _posts/2016-05-12-smr16.md (hunk 9 above)
Comment (DuncanFenning) [inline _posts/2016-05-12-smr16.md]: No worries 👍

**Comment by DuncanFenning** [comment]
Looks good to me! 👍

**Comment by carolinewoodhams** [comment]
Fantastic - let's get this post live! :)

**Pull request merged**